nubia/internal/completion.py

```diff
@@ -204,14 +232,24 @@ def _prepare_args_completions(
                 return []
             # We are completing a value, in this case, we need to get the last
             # meaninful piece of the token `x=[Tr` => `Tr`
+            if isinstance(arg.choices, Callable):
+                choices = arg.choices(
+                    self.cmd.metadata.command.name, subcommand, last_token,
+                    self.doc.text)
+                if not isinstance(choices , List):
+                    print('autocomplete function MUST provide list of strings')
```

Comment:
instead of print, use as the rest of the repository `termcolor.cprint` to display the warning
```suggestion
                    cprint('autocomplete function MUST provide list of strings', 'yellow')
```
Look [here](https://github.com/netenglabs/nubia-cli/blob/dynamic-completion/nubia/internal/nubia.py#L202) for an example

Reply:
Will fix. I need it to be in RED

Reply:
Shouldn't we raise an exception here instead? This is an error of the programmer and it's easier to detect it in tests if this command returns an exception instead of a valid output

Reply:
I agree with Claudio. CHanged it to raise ValueError